### DISCUSSION TITLE
Conversion errors when using 1.4 checkpoint

### DISCUSSION THREAD

**Discussion by carsNcaffeine** [TheLastBen/fast-stable-diffusion#328] [Q&A]
i am trying to train on the 1.4 checkpoint, downloaded from huggingface, then shared from my gdrive.  I am getting conversion errors regardless of whether I use "compatibility mode".  Any idea what I'm doing wrong?

Traceback (most recent call last):
  File "/content/convert_original_stable_diffusion_to_diffusers.py", line 628, in <module>
    checkpoint = torch.load(args.checkpoint_path)["state_dict"]
  File "/usr/local/lib/python3.7/dist-packages/torch/serialization.py", line 712, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/usr/local/lib/python3.7/dist-packages/torch/serialization.py", line 1049, in _load
    result = unpickler.load()
  File "/usr/local/lib/python3.7/dist-packages/torch/serialization.py", line 1042, in find_class
    return super().find_class(mod_name, name)
ModuleNotFoundError: No module named 'pytorch_lightning'
Conversion error, Insufficient RAM or corrupt CKPT, use a 4.7GB CKPT instead of 7GB
Conversion error, Insufficient RAM or corrupt CKPT, use a 4.7GB CKPT instead of 7GB

---------------------------------------------------------------------------

KeyboardInterrupt                         Traceback (most recent call last)

[<ipython-input-6-8254a6f7fa74>](https://localhost:8080/#) in <module>
    119           while not os.path.exists('/content/stable-diffusion-v1-5/unet/diffusion_pytorch_model.bin'):
    120             print('Conversion error, Insufficient RAM or corrupt CKPT, use a 4.7GB CKPT instead of 7GB')
--> 121             time.sleep(5)
    122       else:
    123         while os.path.getsize('/content/model.ckpt') < 1810671599:

KeyboardInterrupt:

**Comment by TheLastBen** [comment]
Try
`!pip install pytorch_lightning`
